src/API/REST/V3/Routes/Donations/RegisterDonationRoutes.php

```diff
@@ -136,6 +137,23 @@ public function registerGetDonations()
                         'default' => 'live',
                         'enum' => ['live', 'test'],
                     ],
+                    'status' => [
+                        'type' => 'string',
+                        'default' => 'any',
+                        'enum' => [
+                            'any',
+                            'publish',
+                            'give_subscription',
+                            'pending',
+                            'processing',
+                            'refunded',
+                            'revoked',
+                            'failed',
+                            'cancelled',
+                            'abandoned',
+                            'preapproval'
+                        ],
+                    ],
```

Comment:
I suggest making status a type of array, that way we can filter by multiple statuses instead of limited to one or all.

You can use campaigns as an example
https://github.com/impress-org/givewp/blob/8b06b4db839d14456ec311a31b448ef7b983e70d/src/API/REST/V3/Routes/Campaigns/RegisterCampaignRoutes.php#L93

Reply:
resolved: a6e6ea55878822703792301f3b86d22447205ba8